domino_cost/cost_dashboard.py

```diff
@@ -42,6 +48,11 @@
     requests_pathname_prefix=requests_pathname_prefix,
 )
 
+billing_tag_display = "block"
```

Comment:
Should these be placed in the `constants.py` file?

Reply:
replaced with different value.